added examples for ISO mDL 18013-5 data model (#233)
oauth-wg/oauth-selective-disclosure-jwt: draft-ietf-oauth-selective-disclosure-jwt.md

{{examples/w3c-vc/disclosures.md}}

## Example 5 - ISO/IEC 18013-5 mDL

This example illustrates how the artifacts defined in this specification can
be used to express an mDL data model defined in ISO/IEC 18013-5 [@ISO.18013-5] encoded as JSON.

> SD-JWT is equivalent to an `issuerSigned` object as defined in [@ISO.18013-5].

Reviewer: ```suggestion
An SD-JWT is equivalent to an `issuerSigned` object as defined in [@ISO.18013-5].
```

> A Combined Format for Presentation with a Holder Binding JWT is equivalent to a Document

Reviewer 2: ```suggestion
An SD-JWT Presentation with a Key Binding JWT is equivalent to a Document
```

structure that contains `docType`, `issuerSigned` object and `deviceSigned` object
as defined in [@ISO.18013-5].

> In this example, Holder Binding is applied and a Combined Format for Presentation is signed
> using a raw public key passed in a `cnf` Claim in the SD-JWT.

Reviewer 2: ```suggestion
In this example, Key Binding is applied and the Key Binding JWT is signed with the private key corresponding to
the raw public key passed in a `cnf` Claim in the SD-JWT.
```

Below is a non-normative example of an SD-JWT expressing an mDL data model of [@ISO.18013-5].

The following data will be used in this example:

<{{examples/18013-5/user_claims.json}}

The payload of a corresponding SD-JWT looks as follows:

<{{examples/18013-5/sd_jwt_payload.json}}

Disclosures:

{{examples/18013-5/disclosures.md}}

> A Combined Format for Presentation for the SD-JWT that discloses only `nameSpace`, `issuing_authority`, and `issuing_country`, and has a Holder Binding JWT, could look as follows:

Reviewer 2: ```suggestion
An SD-JWT Presentation that discloses only `nameSpace`, `issuing_authority`, and `issuing_country`, and has a Key Binding JWT, could look as follows:
```

> A Combined Format for Presentation for the SD-JWT that discloses only discloses `nameSpace`, `issuing_authority` and `issuing_country`, and has a Holder Binding JWT could look as follows:

Reviewer: ```suggestion
A Combined Format for Presentation for the SD-JWT that discloses only `nameSpace`, `issuing_authority`, and `issuing_country`, and has a Holder Binding JWT, could look as follows:
```

> <{{examples/18013-5/combined_presentation.txt}}

Reviewer 2: ```suggestion
<{{examples/18013-5/sd_jwt_presentation.txt}}
```
